src/Streamly/Internal/Data/Parser/ParserK/Type.hs

```diff
@@ -206,67 +106,6 @@ parserDone st (Done _ _) =
     error $ "Bug: fromParserK: inside alternative: " ++ show st
 parserDone _ (Error e) = return $ Failed e
```

Comment:
Why is this function still here? It is not used in this file, it should move along with the code in which it was used.